swagger.yaml

```diff
@@ -31,6 +31,10 @@ paths:
           description: "Successfully returned a cache time for a given id"
           schema:
             $ref: "#/definitions/CacheTime"
+        404:
+          description: "No cachetime was found using the id provided"
```

Comment:
We need more consistency with our names. In this file we have used different variants: cache, cachetime, cache-time, cache time, Cache Time... Let's pick one and use only that one.

Reply:
you are right. I have made it more consistent using "cache time" now :)